include/cuco/dynamic_map.cuh

```diff
@@ -44,7 +44,6 @@ namespace cuco {
  *
  * Current limitations:
  * - Requires keys that are Arithmetic
```

Comment:
Is this a requirement specific to dynamic map or an error in the docs?

Reply:
Updated